qutebrowser/config/configtypes.py

```diff
@@ -1996,3 +1996,15 @@ def to_py(
             return urlmatch.UrlPattern(value)
         except urlmatch.ParseError as e:
             raise configexc.ValidationError(value, str(e))
+
+
+class StatusbarWidget(String):
```

Comment:
Since we don't need all the other validations provided by `String`, maybe it'd be easier to base this on `BaseType` instead? Then you could do the custom validation in `to_py`, which seems a bit clearer to me than overriding `_validate_valid_values`.

No hard feelings either way, though.

Reply:
Personally, I like that most of the validation is handled by the superclass. If we inherit from `BaseType` one would either not validate them or need to copy that check. I prefer the current way a little bit, but I can go both ways, too.